project/profiles/capacity4more/modules/c4m/message/c4m_message/c4m_message.module

```diff
@@ -3234,9 +3240,17 @@ function c4m_message__update__node($node, $account = NULL) {
     $account = user_load($node->uid);
   }
 
+  // Do not create/update message for activity stream, as we deal with
```

Comment:
move this to be the first condition to check, there is no point to load a user if we return early ...